src/com/projectkorra/projectkorra/util/FlightHandler.java

```diff
@@ -199,6 +199,14 @@ public Player getPlayer() {
 		public Player getSource() {
 			return source;
 		}
+		
+		public boolean getCouldFly() {
```

Comment:
All of these changes can go given my previous comments.

Reply:
These should stay regardless. It is now and could be useful in the future.

Reply:
Given the top comment, there's no need a third party would need to access this information. Like how I have createInstance returning a Flight instance where it shouldn't be stored, these methods are asking for people to access them when they don't need to.

Reply:
But there is no harm in them accessing them, since they can't change them.